Ratios/Ratios/View Models/CalculatorViewModel.swift

```diff
@@ -13,9 +13,9 @@ typealias UnitOfMeasurement = Double
 class CalculatorViewModel {
     static func calculateUnitOfMeasurementOfWaterTimes(waterRatio: UnitOfMeasurement, coffee: UnitOfMeasurement, unit: Int) -> UnitOfMeasurement {
         switch unit {
-        case 1:
+        case 1: // Tbsp formula
```

Comment:
Maybe create a separate issue to refactor `unit` to be an enum type so you don't need to add comments for context?